spec/features/admin/products_spec.rb

```diff
@@ -470,16 +470,96 @@
       expect("#{uri.path}?#{uri.query}").to eq spree.admin_product_images_path(product, filter)
     end
 
-    scenario "editing a product's variant unit scale", js: true do
-      product = create(:simple_product, name: 'a product', supplier: @supplier2)
+    context "editing a product's variant unit scale", js: true do
+      let!(:product) { create(:simple_product, name: 'a product', supplier: @supplier2) }
```

Comment:
There are some cases in which you need `let!` but in general it's better to use `let`. It enables us to add test cases which don't need that product or want to create it later.

You are using `product` to visit the page and therefore it will be created before visiting the page which is early enough. You can just use `let` here.